### ISSUE TITLE
Fluid.Sources.Outside allows to specify non-meaningful media

### ISSUE THREAD

**Issue by mwetter** [ibpsa/modelica-ibpsa#1681]
`Fluid.Sources.Outside*` allows through the pull down menu to specify water or glycol. This does not make sense for this model. The annotation should be changed so that only air can be selected.
The same applies to `Fluid.Sources.MassFlowSource_WeatherData`.

This should avoid the confusion reported in https://github.com/OpenModelica/OpenModelica/discussions/9995#discussioncomment-4601211

**Comment by JayHuLBL** [comment]
@mwetter
The `Fluid.Sources.Outside*` and `Fluid.Sources.MassFlowSource_WeatherData` extends the base class `Fluid.Sources.BaseClasses.PartialSource`, which is also used by other sources. If we changed the medium choices in the `PartialSource` to only select air, it will also affect other sources. Two options to avoid it:

1. create a new medium instance `MediumA` and `redeclare final package Medium=MediumA` when extends the base class `PartialSource`. It will be a non-backward compatible change.
2. create a new base class, like `Fluid.Sources.BaseClasses.PartialAirSource`, with medium having only moist air choice. It's backward compatible change.

Option 2 has been implemented in [cbf8d1a](https://github.com/ibpsa/modelica-ibpsa/commit/cbf8d1a3bf26f7924c6d86d9ea64f7c4599e191c).
